examples/hosts/app-integration/schema-upgrade/src/app.tsx

```diff
@@ -0,0 +1,153 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IContainer } from "@fluidframework/container-definitions";
+import { Loader } from "@fluidframework/container-loader";
+
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { TinyliciousService } from "./tinyliciousService";
+import { AppView } from "./appView";
+import { containerKillBitId, InventoryListContainerRuntimeFactory } from "./containerCode";
+import { IContainerKillBit } from "./containerKillBit";
+import { extractStringData, fetchData, applyStringData, writeData } from "./dataHelpers";
+import { IInventoryList } from "./inventoryList";
+
+// In interacting with the service, we need to be explicit about whether we're creating a new document vs. loading
+// an existing one.  We also need to provide the unique ID for the document we are creating or loading from.
+
+// In this app, we'll choose to create a new document when navigating directly to http://localhost:8080.  For the ID,
+// we'll choose to use the current timestamp.  We'll also choose to interpret the URL hash as an existing document's
+// ID to load from, so the URL for a document load will look something like http://localhost:8080/#1596520748752.
+// These policy choices are arbitrary for demo purposes, and can be changed however you'd like.
+let createNew = false;
+if (location.hash.length === 0) {
+    createNew = true;
+    location.hash = Date.now().toString();
+}
+const documentId = location.hash.substring(1);
+document.title = documentId;
+
+async function getInventoryListFromContainer(container: IContainer): Promise<IInventoryList> {
+    // Since we're using a ContainerRuntimeFactoryWithDefaultDataStore, our inventory list is available at the URL "/".
+    const url = "/";
+    const response = await container.request({ url });
+
+    // Verify the response to make sure we got what we expected.
```

Comment:
requestFluidObject<>() can be used here to cut 5 lines of code

Reply:
Same below

Reply:
Done in latest